runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -679,8 +756,44 @@ private void createGameTimer(GameTimer timer)
 		infoBoxManager.addInfoBox(t);
 	}
 
+	private void createGameTimer(GameTimer timer)
+	{
+		createGameTimer(timer, true);
+	}
+
 	private void removeGameTimer(GameTimer timer)
 	{
-		infoBoxManager.removeIf(t -> t instanceof TimerTimer && ((TimerTimer) t).getTimer() == timer);
+		removeGameTimer(timer, NO_TIME_PASSED_THRESHOLD);
+	}
+
+	/**
+	 * Adds a game timer to the infobox manager to render. Accepts a `replaceExisting` option to
```

Comment:
This doc also looks wrong... `removeGameTimer` doesn't add game timers, and doesn't accept a `replaceExisting` option.